template/base/scripts/program/dump.mjs

```diff
@@ -10,20 +11,26 @@ const rpc = process.env.RPC ?? 'https://api.mainnet-beta.solana.com';
 const outputDir = getExternalProgramOutputDir();
 await dump();
 
-/** Dump external programs binaries if needed. */
+/** Dump external programs binaries and accounts if needed. */
 async function dump() {
-  // Ensure we have some external programs to dump.
-  const addresses = getExternalProgramAddresses();
-  if (addresses.length === 0) return;
+  // Ensure we have some external accounts to dump.
+  const programs = getExternalProgramAddresses();
+  const accounts = getExternalAccountAddresses();
+  const binaries = [
+    ...programs.map((program) => `${program}.so`),
+    ...accounts.map((account) => `${account}.json`),
+  ].flat();
```

Comment:
I believe this flat does nothing since you're already de-structuring the arrays.